base/server/src/main/java/org/dogtagpki/server/rest/v2/ACLFilter.java

```diff
@@ -51,14 +76,37 @@ public abstract class ACLFilter extends HttpFilter {
     private static final String LOGGING_MISSING_ACL_MAPPING = "ACL mapping not found; OK";
     private static final String LOGGING_INVALID_ACL_MAPPING = "internal error: invalid ACL mapping";
     private Properties aclProperties;
-    private String acl;
+    private String defaultAcl;
+    private Map<String, String> aclMap;
 
     @Override
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
-
+        String method;
+        String path;
+        String acl = defaultAcl;
         if(request instanceof HttpServletRequest req &&
                 response instanceof HttpServletResponse resp) {
+            method = req.getMethod();
+            path = req.getPathInfo() != null ? req.getPathInfo() : "";
+            final String aclSearch = method + ":" + path;
+            if (aclMap!=null) {
+                String[] aclKeys = aclMap.keySet().stream().filter( key -> {
+                    String keyRegex = key.replaceFirst("\\*", ".*").replace("\\{\\}", "([A-Za-z0-9_\\-]*)");
+                    return aclSearch.matches(keyRegex);
+                } ).toArray(String[]::new);
+                for (String a: aclKeys) {
+                    String[] keyParts = a.split(":");
+                    if (keyParts[0].equals("*")){
+                        acl = aclMap.get(a);
```

Comment:
Do we need a `break` here?

Reply:
Actually it is not needed. In the list we could have an element with "*" and a matching path, followed by an entry with matching method and matching path. If we `break` here the more specific entry will not be selected.